### ISSUE TITLE
`PL_SSL_KEY` or `PL_SSL_PRIVKEY`

### ISSUE THREAD

**Issue by detobel36** [padloc/padloc#283]
For self-host, the page mention the two key words: `PL_SSL_KEY` and `PL_SSL_PRIVKEY`    
https://padloc.app/help/guides/self-host/

Which one is the good one ?

Linked to #254

**Comment by MaKleSoft** [comment]
Good catch! It actually doesn't really matter as long as it matches the variable name used in the `docker-compose.yml` but it should definitely be consistent. I've updated the guide to consistently use `PL_SSL_KEY`
